src/js/user-profile/components/AuthApplicationSection.jsx

```diff
@@ -12,6 +14,49 @@ class AuthApplicationSection extends React.Component {
     let content;
     const verifiedAccountType = 3;// verified ID.me accounts are type 3
     const isVerifiedUser = this.props.userProfile.accountType === verifiedAccountType;
+    const savedForms = this.props.userProfile.savedForms;
+    const hasSavedForms = !!savedForms;
+    
+    //TODO:  (ceh) verify edu and burial form codes
+    const formTitles = {
```

Comment:
Thanks also @jbalboni, combined this makes for a great belt and suspenders approach.